packages/big-design/src/components/List/Item/styled.tsx

```diff
@@ -69,6 +69,19 @@ export const StyledListItem = styled.li<ListItemProps<unknown>>`
   }
 `;
 
+export const StyledLabel = styled.span<any>`
```

Comment:
Is this `any` necessary?

Reply:
No. Types will be updated. But for now I want to validate the approach. Because in case approach will be changed this code might be removed at all